app/additem.tsx

```diff
@@ -0,0 +1,191 @@
+import { useGlobalSearchParams, useRouter } from "expo-router";
+import React, { useState } from "react";
+import {StyleSheet, Text, TextInput, TouchableWithoutFeedback, Keyboard, KeyboardAvoidingView, Platform, ScrollView, Alert, View, TouchableOpacity,} from "react-native";
+import RNDateTimePicker from "@react-native-community/datetimepicker";
+
+const AddItem = () => {
+    const router = useRouter();
+    const { day, date } = useGlobalSearchParams();
+    const [formData, setFormData] = useState({
+        label: "",
+        description: "",
+        startTime: new Date(0, 0, 0, 0, 0),
+        endTime: new Date(0, 0, 0, 23, 59),
+    });
```

Comment:
This useState would benefit from having a type with the four attributes which would allow typescript to infer types and make the page type safe.

Reply:
I added an interface. I think it should be good enough, right?